www/install/steps/process/createDbUser.php

```diff
@@ -63,7 +63,12 @@
 $host = "localhost";
 // if database server is not on localhost...
 if ($parameters['address'] != "127.0.0.1" && $parameters['address'] != "localhost") {
-    $host = $_SERVER['SERVER_ADDR'];
+    $getIpQuery = $link->prepare(
+        'SELECT host FROM information_schema.processlist WHERE ID=connection_id()'
```

Comment:
```suggestion
        'SELECT host FROM information_schema.processlist WHERE ID = connection_id()'
```

Reply:
Hi Igcosta, 
Please, can you add spaces before and after the equal. As, now, sonar broke every job on jenkins when new coding style issue are added.